File: javascript/nivel 3 listas textos/18.js
/*
Ejercicio 18
Comprobar si una palabra es palíndromo
*/
let palabra = prompt("ingrese una palabra: ");

if(palabra === null || palabra.trim() ===""){
  console.log("no se ingreso una palabra");
}else{
  palabra = palabra.toLowerCase();

  let palabraInvertida = palabra.split("").reverse().join("");

  if(palabra === palabraInvertida){
    console.log("la palabra es un palindromo");
  }else{
    console.log("la palabra no es un palindromo");
  }
}

File: python/nivel 3 listas textos/18.py
"""
Ejercicio 18
Comprobar si una palabra es palíndromo
"""
palabra = input("Ingrese una palabra: ")

palabra_invertida = palabra[::-1]

if palabra == palabra_invertida:
  print("la palabra ",palabra," es un palindromo")
else:
  print("la palabra ",palabra," no es un palindromo")
